index.js

```diff
@@ -169,15 +169,21 @@ Choo.prototype.start = function () {
 
 Choo.prototype.mount = function mount (selector) {
   assert.equal(typeof window, 'object', 'choo.mount: window was not found. .mount() must be called in a browser, use .toString() if running in Node')
-  assert.equal(typeof selector, 'string', 'choo.mount: selector should be type string')
+  assert.ok(typeof selector === 'string' || typeof selector === 'object', 'choo.mount: selector should be type String or HTMLElement')
 
   var self = this
 
   documentReady(function () {
     var renderTiming = nanotiming('choo.render')
     var newTree = self.start()
+    if (typeof selector === 'string') {
+      self._tree = document.querySelector(selector)
+    } else {
+      if (selector instanceof window.HTMLElement) {
```

Comment:
In this case, I think it's safe to assume that if it isn't a string, it'll be an HTML element. People would need to try hard to make it fail here I think.

Sorry for being a bit strict here; just out of experience it's that these type of constructions don't compress really well. If file size didn't matter I'd be all for having the stricter check. Thanks!

Reply:
ok :)